Packs/FeedProofpoint/Integrations/FeedProofpoint/FeedProofpoint.py

```diff
@@ -87,6 +87,10 @@ def _build_iterator(
                 # add type/value to item.
                 if "domain" in item:
                     item["type"] = FeedIndicatorType.Domain
+                    # As part of the domain feed, also DomainGlob indicators will be returned, so we are checking if the
+                    # domain has '*' in their value
+                    if '*' in str(item.get("domain")):
+                        item["type"] = FeedIndicatorType.DomainGlob
                     item["value"] = item.get("domain")
```

Comment:
```suggestion
                    ioc_value = item.get("domain", "")
                    if '*' in ioc_value:
                        item["type"] = FeedIndicatorType.DomainGlob
                    item["value"] = ioc_value
```
Shouldn't the value always be a string? I don't think there is a need to explicitly cast to a `str`.

Reply:
The str cast is needed it sometimes fails in unit tests

Reply:
Sababushka - so leave that in. You can still implement the rest of the suggested change.